adapters/utils/runAdapter.ts

```diff
@@ -17,6 +17,7 @@ function getUnixTimeNow() {
 export default async function runAdapter(volumeAdapter: BaseAdapter, cleanCurrentDayTimestamp: number, chainBlocks: ChainBlocks, id?: string, version?: string, {
   adapterVersion = 1,
   isTest = false,
+  prefetch = null,
```

Comment:
volumeAdapter will have the prefetch field right? if you use that instead of the extra field here, prefetch will work without automatically for all the code calling runAdapter without any modification to them